x/ccv/provider/keeper/keeper_test.go

```diff
@@ -249,7 +249,7 @@ func (suite *KeeperTestSuite) TestHandleSlashPacketDoubleSigning() {
 }
 
 func (suite *KeeperTestSuite) TestHandleSlashPacketErrors() {
-	providerStakingKeeper := suite.providerChain.App.GetStakingKeeper()
+	providerStakingKeeper := suite.providerChain.App.(*appProvider.App).StakingKeeper
```

Comment:
Why not use the `GetStakingKeeper` function defined in `app.go`?

Reply:
Because in order to have `GetStakingKeeper()` returns the `ConsumerKeeper`, we changed its function signature to return an IBC staking keeper interface rather than a concrete type. However this interface has only two methods defined: `GetHistoricalInfo()` and
`UnbondingTime()`. It might be improved.